TrelloAutotest/Pages/TrelloBoardPage.cs

```diff
@@ -1,45 +1,56 @@
-﻿using OpenQA.Selenium;
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
 using System.Linq;
 using TrelloAutotest.Messages;
+using TrelloAutotest.Selectors;
 using TrelloAutotests.Pages;
 
 namespace TrelloAutotest.Pages
 {
-    using Selectors = Selectors.BoardSelectors;
-    using BaseSelectors = Selectors.BaseSelectors;
-
     public class TrelloBoardPage : BasePage
     {
-        public TrelloBoardPage() => Wait.Until(d => driver.FindElements(Selectors.BoardHeader).Any());
-
         public void OpenBoardMenu() => driver.FindElements(By.TagName("a")).First(x => x.Text.Contains(MessageText.ShowMenu)).Click();
-        
-        public bool IsLoginCorrect() => driver.FindElements(Selectors.BoardHeader).Any();
 
-        public bool IsUrlCorrect(string BoardName)
+       
+
+        public TrelloBoardPage IsUrlCorrect(string BoardName)
         {
-            Wait.Until(d => driver.FindElements(Selectors.BoardViewButton).Any());
+            Wait.Until(d => driver.FindElements(BoardSelectors.BoardViewButton).Any());
 
             var SplitName = BoardName.Contains(' ') ? BoardName.ToLower().Replace(' ', '-').TrimEnd('-') : BoardName.ToLower();
             var url = TrimUrl(driver.Url);
 
-            return url == (SplitName);
+            Assert.IsTrue(url == (SplitName));
+            return this;
         }
 
         public string TrimUrl(string Url) => Url.Split('/').Last();
```

Comment:
1. If this function is public, it should return a page. There is no need for having this function public.
2. The name is misleading - trimming in the programming is different from what you're doing in this function.